docs/how-to-guides/develop-a-tool/add-a-tool-icon.md

```diff
@@ -20,10 +20,11 @@ Adding a custom tool icon is optional. If you do not provide one, the system use
   pip install pillow
   ```
 
-## Add tool icon with _icon_ parameter 
+## Add tool icon using command
+### Add a tool icon for both light and dark mode  
```

Comment:
I think we still keep the singular icon path to ensure compatibility with existing tools. But not sure we still need to provide this approach in our doc. Maybe we can only keep the light&dark way. Please confirm with jincheng.

Reply:
agree.

Reply:
Moved light&dark to FAQ, thanks.